content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -396,49 +410,9 @@ to restrict access to only the appropriate callers.
     caption="A service mesh with two namespaces and an exposed service"
     >}}
 
-When using [multiple clusters](#multiple-clusters), the namespaces in each
-cluster sharing the same name are considered the same namespace. For example,
-`Service B` in the `foo` namespace of `cluster-1` and `Service B` in the
-`foo` namespace of `cluster-2` refer to the same service, and Istio merges their
-endpoints for service discovery and load balancing.
-
-{{< image width="50%"
-    link="cluster-ns.svg"
-    alt="A service mesh with two clusters with the same namespace"
-    title="Multicluster namespaces"
-    caption="A service mesh with clusters with the same namespace"
-    >}}
-
-### Cluster tenancy
-
-Istio supports using clusters as a unit of tenancy. In this case, you can give
-each team a dedicated cluster or set of clusters to deploy their
-workloads. Permissions for a cluster are usually limited to the members of the
-team that owns it. You can set various roles for finer grained control, for
-example:
-
-- Cluster administrator
-- Developer
-
-To use cluster tenancy with Istio, you configure each cluster as an independent
-mesh. Alternatively, you can use Istio to implement a group of clusters as a
-single tenant. Then, each team can own one or more clusters, but you configure
-all their clusters as a single mesh. To connect the meshes of the various teams
-together, you can federate the meshes into a multi-mesh deployment.
-
-{{< image width="50%"
-    link="cluster-iso.svg"
-    alt="Two isolated service meshes with two clusters and two namespaces"
-    title="Cluster isolation"
-    caption="Two isolated service meshes with two clusters and two namespaces"
-    >}}
-
-Since a different team or organization operates each mesh, service naming
-is rarely distinct. For example, the `mysvc` in the `foo` namespace of
-`cluster-1` and the `mysvc` service in the `foo` namespace of
-`cluster-2` do not refer to the same service. The most common example is the
-scenario in Kubernetes where many teams deploy their workloads to the `default`
-namespace.
+### Cluster isolation
```

Comment:
This used to be more about using meshes as tenancy (it was probably mistitled). 

I think we want to do more work to make cluster tenancy actually work well, and also think we should talk about using multiple meshes as tenancy.

Reply:
what does it even mean? and why does it matter to a single Istio mesh? I mean the persona doing this is viewing the doc from the perspective of a single mesh, based on what is possible today in Istio. We dont have examples/guides for multi-mesh or guides explaining how to do trust bundle updates, replication, etc.

Reply:
The target audience is a platform admin. They need to know how to deal with tenancy within their platform. One option is cluster tenancy, which doesn't quite work right by being overly tied to control plane deployments, but is something we should work on improving. Another option is mesh tenancy. We should discuss these models here, if we're going to discuss one model that doesn't quite work right we should discuss the other too :)

Reply:
Okay changed to cluster tenancy and also added mesh tenancy